bot/src/main/java/com/almightyalpaca/discord/jdabutler/commands/commands/NotifyCommand.java

```diff
@@ -339,6 +340,16 @@ private static TextChannel getBlacklistChannel()
     }
 
     private static void respond(Message origMsg, String newMessageContent)
+    {
+        respond(origMsg, new MessageBuilder(newMessageContent).build());
+    }
+
+    private static void respond(Message origMsg, MessageEmbed newMessageContent)
+    {
+        respond(origMsg, new MessageBuilder(newMessageContent).build());
+    }
+
+    private static void respond(Message origMsg, Message newMessageContent)
```

Comment:
These 2 additional ones needed anymore? Consiering that they are only on the specific command and not part of a base class, i don't really see how this is super helpful